h/static/styles/partials/_form-input.scss

```diff
@@ -173,7 +192,11 @@
     position: relative;
     left: -1px;
     top: -1px;
+
+    // Assume largest possible padding if CSS vars are not supported.
     padding-bottom: $bottom-padding - 2px;
+    padding-bottom: calc(var(--bottom-padding) - 2px);
```

Comment:
IE 11 is the only browser we support that doesn't support CSS vars. It will ignore this second `padding-bottom` declaration and just use a bit more padding than necessary.